README.md

```diff
@@ -16,13 +16,5 @@
 
 ### Some endpoints to check out 
 
-#### GET requests
-
--   `http://localhost:8080/api/transcripts`
--   `http://localhost:8080/api/users`
-
-### TODOS:
-- // Add swagger api documentation
-- // Set up the application on AWS
-- // Supply more meaningful data in the init.sql file
+- Access the endpoints documentation here: `http://localhost:8080/api/docs`
```

Comment:
Tried to access at 8080/api/docs and nothing loaded.

<img width="352" alt="Screen Shot 2023-03-10 at 6 59 33 PM" src="https://user-images.githubusercontent.com/755825/224451040-527dde95-3427-4423-b07d-c9b02e10151b.png">

Reply:
I can see the docs
<img width="1042" alt="Screenshot 2023-03-13 at 8 38 24 PM" src="https://user-images.githubusercontent.com/62019510/224814299-02c5a888-9124-4f65-9243-6b2ee43c8102.png">